sark/code/line.py

```diff
@@ -180,9 +180,19 @@ def endEA(self):
     def disasm(self):
         """Line Disassembly"""
         return idc.GetDisasm(self.ea)
+    @property
+    def type(self):
```

Comment:
Is this meant for text-output only, or do you have other uses for this property?